### ISSUE TITLE
Automatic web link for published projects

### ISSUE THREAD

**Issue by brenocq** [brenocq/atta#20]
## Feature description
After #17 it is possible to execute projects in the browser. It would be interesting to automatically generate a link to execute the latest version of a published project.

## Steps to publish
1. Create github repo, clone, program atta project
2. Update README.md comment (title, description, image link, web build link)
3. `atta --create-workflow-web-publisher`
4. `atta --create-workflow-test-build` (why not, right?)
5. Push changes to github
6. `atta --publish` (or manually create a issue in `https://github.com/brenocq-atta/projects`)

**Why readme comment?**
This is necessary because published projects receive a page like `https://atta.brenocq.com/projects/<user>/<repo>`. The readme comment is parsed to display the project info in the page.

**Where can I access the published project web build?**
The web build will be available at: `https://atta.brenocq.com/projects/<user>/<repo>/build`

## Big picture
<p align="center">
<img  src="https://user-images.githubusercontent.com/17342434/174404060-bf83e740-1784-4548-b337-592e16edf3ba.jpg" />
</p>

## Tasks
- [x] (Atta projects repo) Automatically update [published projects json](https://github.com/brenocq-atta/projects/blob/main/projects.json) from closed issue.
- [x] (Atta website) Run project web build from google storage files
- [x] (Published repo) Find alternative to metadata (ugly readme) -> html comment is the way to go
- [x] (Atta website) Fix README.md parser to get data from comment, instead of metadata
- [x] (Published repo) Create workflow to create web build and publish to google storage
- [x] (Atta website) Make `https://atta.brenocq.com/projects/<user>/<repo>/build` available
  - [x] Parse web build link from README.md 
  - [x] Improve UI to show project build in the project page

## Postponed to the future
- (Atta) Create `atta --create-project <name>` 
- (Atta) Create `atta --create-workflow-web-publisher` to automate workflow creation
- (Atta) Create `atta --create-workflow-test-build` to automate workflow creation
- (Atta) Create `atta --publish` to automate publish process

## Atta command helper

**`atta --create-workflow-web-publisher` description**
   - It will scan the repo looking for atta projects (`.atta` files) and create a `.github/workflows/webPublisher.yml` to publish the web build of each project found
   - Publish the web build = updating web build in the user's google storage (atta.data and atta.js)

**`atta --create-workflow-test-build` description**
   - It will scan the repo looking for atta projects (`.atta` files) and create `.github/workflows/<linux/macos/windows>.yml`
   - The workflow will test if the project build to different operating systems

**`atta --publish` description**
 - It will automatically create an issue in `https://github.com/brenocq-atta/projects`

### Google storage
1. Create project
2. Create bucket
3. Create credentials
4. Create credentials json key
5. Add json key to github secrets
